samples/js/node/notebooks/pose-estimation.nnb

```diff
@@ -0,0 +1,133 @@
+            "source": [
+                "const imgData = await getImageData(imagePath);\n\nconst originalImage = cv.matFromImageData(imgData);\nconst { cols: originalWidth, rows: originalHeight } = originalImage;\n\nconst image = new cv.Mat();\ncv.cvtColor(originalImage, image, cv.COLOR_RGBA2RGB);\ncv.cvtColor(image, image, cv.COLOR_BGR2RGB);\ncv.resize(image, image, new cv.Size(width, height), cv.INTER_AREA);\n\n// NHWC to NCHW\nconst inputImage = transform(image.data, { width, height }, [0, 1, 2]);"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "markdown",
+            "source": [
+                "## Define postprocessing functions"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "typescript",
+            "source": [
+                "\n// Utility function to get the coordinates of the maximum value in a tensor\nfunction getCoords(tensor) {\n  const { values, indices } = tf.topk(tensor.flatten(), 1);\n  const [maxVal] = values.dataSync();\n  const [maxIndex] = indices.dataSync();\n  const x = maxIndex % tensor.shape[1];\n  const y = Math.floor(maxIndex / tensor.shape[1]);\n\n  return { x, y, confidence: maxVal };\n}\n\n// Step 2: Draw keypoints on the input image\nfunction drawKeypoints(image, keypoints) {\n  keypoints.forEach(keypoint => {\n    if (keypoint.confidence > THRESHOLD) {\n      cv.circle(image,\n        new cv.Point(keypoint.x, keypoint.y),\n        2,\n        COLOR,\n        3,\n      );\n    }\n  });\n}\n\n// Step 3: Draw skeleton (lines between keypoints) on the input image\nfunction drawSkeleton(image, keypoints) {\n  const skeleton = [\n    [1, 2], [1, 5], [2, 3], [3, 4], [5, 6], [6, 7], [1, 8], [8, 9], [9, 10],\n    [1, 11], [11, 12], [12, 13], [1, 0], [0, 14], [14, 16], [0, 15], [15, 17],\n  ];\n\n  skeleton.forEach(([start, end]) => {\n    const startPoint = keypoints[start];\n    const endPoint = keypoints[end];\n\n    if (startPoint.confidence > THRESHOLD\n      && endPoint.confidence > THRESHOLD) {\n      cv.line(image,\n        new cv.Point(startPoint.x, startPoint.y),\n        new cv.Point(endPoint.x, endPoint.y),\n        COLOR,\n        2,\n      );\n    }\n  });\n}\n\nfunction toTFTensor(ovTensor) {\n  return tf.tensor(ovTensor.data, ovTensor.getShape());\n}"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "markdown",
+            "source": [
+                "## Do Inference"
+            ],
+            "outputs": []
+        },
+        {
+            "language": "typescript",
+            "source": [
+                "const tensorData = new Float32Array(inputImage);\nconst tensor = new ov.Tensor(ov.element.f32, inputLayer.shape, tensorData);\n\nconst inferRequest = compiledModel.createInferRequest();\ninferRequest.setInputTensor(tensor);\nconst outputs = inferRequest.infer();\n\nconst pafs = outputs[pafsOutputKey];\nconst heatmaps = outputs[heatmapsOutputKey];"
```

Comment:
If I see correctly this model operates on FP16-INT8. Didn't you have any problems with it?